package.json

```diff
@@ -15,33 +15,33 @@
     "lint": "eslint \"src/**/*.svelte\""
   },
   "dependencies": {
-    "compression": "^1.7.1",
+    "compression": "1.7.1",
     "polka": "next",
-    "sirv": "^0.4.0"
+    "sirv": "0.4.0"
   },
   "devDependencies": {
-    "@babel/core": "^7.0.0",
-    "@babel/plugin-syntax-dynamic-import": "^7.0.0",
-    "@babel/plugin-transform-runtime": "^7.0.0",
-    "@babel/preset-env": "^7.0.0",
-    "@babel/runtime": "^7.0.0",
+    "@babel/core": "7.0.0",
+    "@babel/plugin-syntax-dynamic-import": "7.0.0",
+    "@babel/plugin-transform-runtime": "7.0.0",
+    "@babel/preset-env": "7.0.0",
+    "@babel/runtime": "7.0.0",
     "@rollup/plugin-commonjs": "11.0.2",
-    "@rollup/plugin-node-resolve": "^7.0.0",
-    "@rollup/plugin-replace": "^2.2.0",
-    "eslint": "^6.8.0",
-    "eslint-config-airbnb-base": "^14.1.0",
-    "eslint-config-prettier": "^6.11.0",
-    "eslint-plugin-import": "^2.20.2",
-    "eslint-plugin-prettier": "^3.1.3",
-    "eslint-plugin-svelte3": "^2.7.3",
-    "npm-run-all": "^4.1.5",
-    "prettier": "^2.0.5",
-    "prettier-plugin-svelte": "^1.1.0",
-    "rollup": "^2.0.0",
-    "rollup-plugin-babel": "^4.0.2",
-    "rollup-plugin-svelte": "^5.0.1",
-    "rollup-plugin-terser": "^5.3.0",
-    "sapper": "^0.27.0",
-    "svelte": "^3.0.0"
+    "@rollup/plugin-node-resolve": "7.0.0",
+    "@rollup/plugin-replace": "2.2.0",
+    "eslint": "6.8.0",
+    "eslint-config-airbnb-base": "14.1.0",
+    "eslint-config-prettier": "6.11.0",
+    "eslint-plugin-import": "2.20.2",
+    "eslint-plugin-prettier": "3.1.3",
+    "eslint-plugin-svelte3": "2.7.3",
+    "npm-run-all": "4.1.5",
+    "prettier": "2.0.5",
+    "prettier-plugin-svelte": "1.1.0",
+    "rollup": "2.0.0",
+    "rollup-plugin-babel": "4.0.2",
+    "rollup-plugin-svelte": "5.0.1",
+    "rollup-plugin-terser": "5.3.0",
+    "sapper": "^0.27.13",
+    "svelte": "3.0.0"
```

Comment:
By removing the `^`, the dependencies are actually downgraded. Try running `npm outdated` and upgrade the dependencies to latest compatible version.

Reply:
done